CommunityFridgeMapApi/template.yaml

```diff
@@ -7,6 +7,9 @@ Description: >
 
 # More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
 Globals:
+  Api:
+    BinaryMediaTypes:
+      - image/webp
```

Comment:
This is required to have a binary response or request.

Reply:
Does it need to be a global or can it be in `ImageFunction`?

Reply:
To be honest, I'm not entirely sure how to make it specific to `ImageFunction`.

`BinaryMediaTypes` is a property of `AWS::Serverless::Api` resource type ([reference](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-resource-api.html#sam-resource-api-properties)), and I could not find an equivalent property in `AWS::Serverless::Function` type.